model/src/cfd.rs

```diff
@@ -196,7 +196,7 @@ impl From<Origin> for Role {
 pub struct Offer {
     pub id: OfferId,
 
-    pub trading_pair: TradingPair,
+    pub contract_symbol: ContractSymbol,
```

Comment:
🛠 I think this is not only a breaking change in the HTTP API but a breaking change on the network layer! I think we need a `#[serde(rename = "trading_pair")]` here, otherwise already rolled out `0.5.x` won't be compatible.

Reply:
Even though this field is not used, I think we would still run into trouble deserializing the `Offer` if the maker sends it out like this, because the field will be missing on the taker side.

Reply:
good point. We need to use serde rename here indeed. I'll add a fixup.